server/src/server.ts

```diff
@@ -19,6 +19,8 @@ import { Telemetry } from "./telemetry/types";
 import { attachDocumentHooks } from "./services/documents/attachDocumentHooks";
 import { availableVersions } from "./services/initialization/updateAvailableSolcVersions";
 import { onDocumentFormatting } from "./services/formatting/onDocumentFormatting";
+import { onSemanticTokensFull } from "./services/semanticHighlight/onSemanticTokensFull";
+import { onDocumentSymbol } from "./services/documentSymbol/onDocumentSymbol";
 
 export default function setupServer(
```

Comment:
@OmarTawfik tried the node js flags approach but it didn't work. The `report-on-fatalerror` doesn't work, tested with a sample program triggering the panic from slang. 

About the extension logs, it could be possible (althought not sure if viable) to scan for extension logs and try to pattern-match anything related to panics. For testing this, I'd need if you could rebase the `simulate-panic` branch on top of `main` on slang. Because otherwise I can't build the LSP because of some changes in the modules in the latest versions

Reply:
Sure. I just rebased it on top of the `v0.10.1` tag, which is the version you are using. Does that work, or do you want to use `main` instead? it has a few unreleased (yet) changes that shouldn't affect you.

Reply:
I also did some debugging today, and I found out that it helps if you set env variable `RUST_BACKTRACE` to `FULL`. It should print the panic traces in the stderr of the server process.

Reply:
Let's put this aside for now, until we move to Slang `0.11.0`